src/utils/googleAnalytics.ts

```diff
@@ -10,7 +10,7 @@ export const FOOTER_CATEGORY = 'FOOTER_CATEGORY'
 
 const getGoogleAnalyticsTrackingID = () => {
   return typeof window !== 'undefined' &&
-    window.location.hostname.split('.')[0] === 'gnosis-safe'
+    window.location.hostname.split('.')[0] === 'safe'
     ? process.env.GATSBY_GA_MAINNET
     : process.env.GATSBY_GA_RINKEBY
```

Comment:
Please rename these vars to `GATSBY_GA_PROD` and `GATSBY_GA_STAGING`.

Reply:
Are we also going to adjust the github secrets naming?

Reply:
You can ask @luarx to do it.

Reply:
I changed it in the code so we will have to adjust the secrets once this PR is merged.